docs/20_smart-contracts/10_getting-started/20_accounts/index.md

```diff
@@ -0,0 +1,208 @@
+
+Every account has the ability to have one single smart contract deployed to it, however, that smart contract can be updated and replaced at will. 
+
+For more information about deploying smart contracts to your accounts, please see our **DUNE Guide**. 
+
+
+## Creating an Account with DUNE
+
+Once you have **DUNE** set up you can start creating accounts on your local development environment with a single command. 
+
+```
+dune --create-account <ACCOUNT_NAME>
+```
+
+If you want to see the information related to the account you just created, you can use the following command. 
+
+```
+dune -- cleos get account
+```
+
+
+## Ownership of Digital Assets
+
+When something is *owned* by an account on a blockchain, it means that a row in the decentralized database says that the digital asset belongs to that account. 
+
+In most cases, only the specific account that is registered as the owner of that digital asset and has the private key that can prove ownership of the account (via a signature) may manipulate that data. 
+
+Keep in mind that a smart contract also co-owns that data with the user, and might be able to manipulate all data that is stored inside of its tables without the explicit consent of the user.
+
+Smart contract developers may also update smart contracts at will, so contracts that have security or financial implications may relinquish their ability to update their smart contracts in order to trade upgradeability for increased user trust. 
+
+
+## Relinquishing ownership of an account
+
+Upgradeability has significant benefits for smart contract development, but isn’t always wanted. 
+
+In order to relinquish a smart contract’s upgradeability, you have two options.
+
+
+### NULL account
+
+You may set the contract account’s owner and active permissions to `eosio.null@active`. This is a `NULL` account that is specifically designed for these purposes. It has no private key or owner. 
+
+This is a good option if you want to relinquish control of this account **forever**.
+
+
+```
+dune -- cleos set account permission <ACCOUNT> active '{"threshold":1,"keys":[],"accounts":[{"permission":{"actor":"eosio.null","permission":"active"},"weight":1},{"permission":{"actor":"<ACCOUNT>","permission":"eosio.code"},"weight":1}],"waits":[]}' owner -p <ACCOUNT>
+dune -- cleos set account permission <ACCOUNT> owner '{"threshold": 1, "keys":[], "accounts":[{"permission":{"actor":"eosio.null","permission":"active"},"weight":1}], "waits":[] }' -p <ACCOUNT>@owner
+```
+
+(note the eosio.code addition for the active permission, which you might need if the account is a smart contract!)
+
+### Prods accounts
+
+Alternatively, you may set the contract account’s owner and active permissions to three different producer-controller (network consensus-controlled) accounts so that if there is ever an issue with this contract you can request the help of the producers to upgrade the contract. 

```

Comment:
having this option listed first begs the questions: why three? why not two or four?
so if it is listed the last it is clearer that this is yet another simpler alternative to the others.

Reply:
Not sure I understand.

There are only three accounts that you can relinquish control to which are controlled by prods: eosio.prods, prod.major, and prod.minor

Reply:
aaahhh I understand now. three `producer-controller accounts` NOT `three accounts` as I red it the first time. great!